lib/api/webSockets.js

```diff
@@ -70,6 +70,10 @@ function handleResponse(msg) {
 			}); // resolve chain requests
 		} else if (res.result === 'success') {
 			req.resolve(res);
+		} else if (res.error && res.error.match(/device/igm)) {
```

Comment:
I don't like detecting error type by searching keywords. How does `res` look? Is there some code? There should be at least a definite way to separate Suitest-produced error from network error.

Reply:
@andriichumak  response is
```
{
  "messageId": "50b31530-4938-11e8-a6b0-15fec0ed97af",
  "content": {
    "result": "error",
    "error": "Device is disabled"
  }
}
```

Reply:
OK, then make it SuitestError.UNKNOWN_ERROR with message from "Device is disabled".
WS error must appear only when network problem occurs. E.g. on "error" event in webSocket.js file.